apps/web/components/apps/CalendarListContainer.tsx

```diff
@@ -168,7 +168,11 @@ function ConnectedCalendarsList(props: Props) {
                     slug={item.integration.slug}
                     title={item.integration.title}
                     logo={item.integration.logo}
-                    description={item.primary?.externalId || "No external Id"}
+                    description={
+                      (item.primary?.integration === "office365_calendar"
+                        ? item.primary?.email
+                        : item.primary?.externalId) ?? "No external Id"
```

Comment:
Do we wanna show just an untranslated english string: "No external Id" here? we can also just not show a description if there is none

Reply:
yeah we can show description but user won't be able to differentiate between multiple account ( like gmail).

Reply:
Can we move this logic to the app specific `listCalendars` call? I think this will help prevent future tech debt.